docs/semgrep-code/policies.md

```diff
@@ -0,0 +1,116 @@
+                <li>Ruleset</li>
+                <li>Language</li>
+            </ul>
+            These filters are explained further below as they correspond to the Rule pane columns also.
+        </dd>
+    <dt>Rule pane</dt>
+        <dd>
+            The right pane displays a number of rules with the <b>Matching Rules</b> title. This pane visualizes the rules that are used in your Semgrep Cloud Platform organization and allows you to edit their assigned modes (Monitor, Comment, Block, and Disabled). You can make these edits either one by one or through bulk editing of many rules. You can also use the <b>Search for rule names or ids</b> The individual columns are explained separately below in the <a href="#rule-pane-in-detail">Rule pane in detail</a> section.
+        </dd>
+</dl>
+
+### Rule pane in detail
+
+This section explains columns in the rule pane of the Policies page in detail:
+
+- **Rule name**: Name of the rule that Semgrep Code uses for scanning.
+- **Severity**: The higher the severity, the more critical issues can a rule potentially detect. The Policies page displays the **high**, **medium**, and **low** severities.
+- **Confidence**: Indicates confidence of the rule to detect true positives. There are rules with **high**, **medium**, and **low** confidence.
+- **Source**: Indicates whether the rule is a **Pro**, **Community rule**, or a **Custom rule** .
+    - **Pro**: Authored by Semgrep with cross-file (interfile), and cross-function (interprocedural) analysis capabilities providing you with enhanced scan accuracy. For more information, see [Pro rules](/semgrep-code/pro-rules/) documentation.
+    - **Community**: Authored by Semgrep, Inc or external contributors such as Trail of Bits.
+    - **Custom**: Rules created within your Semgrep organization. For more information, see [Private rules](/writing-rules/private-rules/) documentation.
+- **Ruleset**: Rules are also organized in rulesets. Rulesets are groups of rules related through a programming language, OWASP category, or framework.
+- **Mode**: Specifies an action that a rule triggers when a rule detects a finding (sometimes called matching - when a rule matches a code). The mode behavior in which the finding is reported to you and your developers. There are the three following modes:
+    - **Monitor**: Display findings only on the [Findings](https://semgrep.dev/orgs/-/findings?tab=open) page of Semgrep Code.
+    - **Comment**: Display findings on the [Findings](https://semgrep.dev/orgs/-/findings?tab=open) page of Semgrep Code and create comments in MRs or PRs.
+    - **Block**: Display findings on the [Findings](https://semgrep.dev/orgs/-/findings?tab=open) page of Semgrep Code, create comments in MRs or PRs, and block PRs or MRs where the finding was detected.
+    - **Disabled**: Disable rules, so they can no longer detect findings. This is helpful if rules are too noisy and detect many false positives that you otherwise need to manually ignore.
```

Comment:
```suggestion
    - **Disabled**: Disabled rules will no longer scan your code or detect findings. This is helpful if rules are too noisy and detect many false positives that you otherwise need to manually ignore.
```